sdks/python/tox.ini

```diff
@@ -27,6 +27,7 @@ select = E3
 deps=
   pep8
   pylint
+  pyhamcrest
```

Comment:
I suggest keeping it in both places and adding this to test_requires in setup.py.  Unfortunately, this doesn't interact well with pip (https://github.com/pypa/pip/issues/1197), but semantically, that is the correct place to put it and it will make python setup.py test work.